springfield/firefox/templates/firefox/landing/win10-eos.html

```diff
@@ -0,0 +1,187 @@
+  <!-- CTA section -->
+  {% set cta_title = 'Like the new Firefox? Take it with you.' %}
+  {% set cta_body = 'Sync your bookmarks, passwords, and settings so Firefox feels familiar wherever you sign in.' %}
+  {% set cta_action = 'Get started' %}
+{% endif %}
+
+{% block page_title %}{{ main_title }}{% endblock %}
+{% block page_desc %}{% endblock %}
+
+{% block page_css %}
+  {{ css_bundle('protocol-split') }}
+  {{ css_bundle('firefox_win10_eos') }}
+{% endblock %}
+
+{% block site_header %}{% endblock %}
+
+{% block content %}
+<main class="mzp-t-firefox">
+  <div class="c-logo-wrapper mzp-l-content mzp-t-content-max">
+    <div class="c-logo-image"></div>
+  </div>
+  <div class="mzp-l-content mzp-t-content-max">
+    <h1 class="c-main-title">{{ main_title }}</h1>
+    <div class="c-box-wrapper mzp-l-columns mzp-t-columns-three">
+      <section class="c-box">
+        <div class="c-box-video mzp-c-video">
+          <video
+            preload="none"
+            loop
+            controls
+            width="1040"
+            height="585"
+            poster="{{ static(box_one_poster_src) }}">
+              <source src="{{ box_one_video_src }}" type="video/webm">
+          </video>
+        </div>
+        <h2 class="c-box-title">{{ box_one_title }}</h2>
+        <p class="c-box-body">{{ box_one_body }}</p>
+      </section>
+      <section class="c-box">
+        <div class="c-box-video mzp-c-video">
+          <video
+            preload="none"
+            loop
+            controls
+            width="1136"
+            height="641"
+            poster="{{ static(box_two_poster_src) }}">
+              <source src="{{ box_two_video_src }}" type="video/webm">
+          </video>
+        </div>
+        <h2 class="c-box-title">{{ box_two_title }}</h2>
+        <p class="c-box-body">{{ box_two_body }}</p>
+      </section>
+      <section class="c-box">
+        <div class="c-box-video mzp-c-video">
+          <video
+            preload="none"
+            loop
+            controls
+            width="847"
+            height="476"
+            poster="{{ static('img/firefox/landing/win10-eos/new-tab/poster-high-res.jpg') }}">
+              <source src="{{ box_three_video_src }}" type="video/webm">
+          </video>
+        </div>
+        <h2 class="c-box-title">{{ box_three_title }}</h2>
+        <p class="c-box-body">{{ box_three_body | safe | format(icon_title) }}</p>
+      </section>
+    </div>
+    <section class="c-cta mzp-c-split mzp-l-split-reversed mzp-l-split-body-wide">
+      <div class="mzp-c-split-container">
+        <div class="mzp-c-split-body ">
+          <h2 class="c-cta-title">{{ cta_title | safe }}</h2>
+          <p class="c-cta-body">{{ cta_body }}</p>
+          <!-- TODO: check attribution & confirm UTM params -->
+          {{ fxa_button(
+            entrypoint='www.firefox.com-win10-eos-page',
+            button_text=cta_action,
+            optional_parameters={'utm_campaign': 'win10-eos'},
```

Comment:
TODO: Is the TODO resolved?

Q: should the entrypoint match the updated URL?

(Only URL was to change, other ids like the campaign bits can stay as "win10-eos" et al.?)